contributing_to_docs/doc_guidelines.adoc

```diff
@@ -61,7 +61,8 @@ would be appropriate for a topic titled Managing Authorization Policies.
 a|- Use title case in all topic titles and section headings. See
 http://titlecase.com/ for more information.
 - Try to be as descriptive as possible with the topic title or section headings
-without making them unnecessarily too long.
+without making them unnecessarily too long. Use a verb form in headings (for example,
+Creating, Managing, Using, etc.) whenever possible.
```

Comment:
I like the idea behind "Understanding". I think a lot of topics could use that and it might be a good idea for a house for the reference type stuff we need to include sometimes.

There might be a danger of overlap between "Understanding" sections and the "Overview" section. Should we be defining what should go into an "Overview" section somewhere? (Do we do that somewhere already? I should really check before I make this comment...)